src/Misc/Poly.jl

```diff
@@ -1060,3 +1060,15 @@ function gcd_right(f::NCPolyRingElem, g::NCPolyRingElem)
   end
   return f
 end
+
+##
+
+function numerator(f::QQPolyRingElem, parent::ZZPolyRing = Hecke.Globals.Zx)
```

Comment:
Should we maybe also introduce a `Nemo.Globals.Zx` etc.? Then we could move this and a couple other similar things to Nemo?

CC @lgoettgens